src/main/java/com/rocketden/main/dto/game/GameTimerMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.rocketden.main.dto.game;
+
+import com.rocketden.main.game_object.GameTimer;
+
+public class GameTimerMapper {
```

Comment:
Could you add a GameTimerMapperTests class?

Reply:
Done!